org.mwc.cmap.legacy/src/MWC/GUI/Editable.java

```diff
@@ -397,8 +397,8 @@ public static void testTheseParameters(final Editable toBeTested)
           final int len = pd.length;
           if (len == 0)
           {
-            System.out.println("zero property editors found for " + toBeTested
```

Comment:
@saulhidalgoaular  the error logging in this file is needed - it helps us track badly configured editors (such as when we don't provide the getter/setter for a named property)

So, we should keep the `system.out` entries in this file.